dapr/clients/grpc/_response.py

```diff
@@ -671,23 +672,32 @@ def __init__(self):
     def get_items(self):
         return self.items
 
-    def watch_configuration(self, stub, store_name, keys, config_metadata):
+    def watch_configuration(self, stub: api_service_v1.DaprStub, store_name: str,
```

Comment:
In the Java SDK, we have created a new interface to keep Alpha APIs separate. Making it clear that they can break.